### ISSUE TITLE
Add training to client

### ISSUE THREAD

**Issue by bourdakos1** [cloud-annotations/training#150]
![not_bound_to_training_service](https://user-images.githubusercontent.com/4212769/71691064-63150680-2d74-11ea-8f48-03ffcd724a23.png)

**Comment by bourdakos1** [comment]
example loss:
```
loss = 2.8680978, step = 118 (1.824 sec)
```

**Comment by bourdakos1** [comment]
TODO:
- [ ] show deactivate train button when loading instances
- [ ] show create WML instance notification if none available
- [ ] implement download models
- [ ] use skeletons instead of `loading...`
- [ ] show create WML on training page if none available
